test/functional/wallet_backup.py

```diff
@@ -246,13 +232,13 @@ def run_test(self):
 
         # Backup to source wallet file must fail
         sourcePaths = [
-            os.path.join(self.nodes[0].wallets_path, self.default_wallet_name, self.wallet_data_filename),
-            os.path.join(self.nodes[0].wallets_path, '.', self.default_wallet_name, self.wallet_data_filename),
-            os.path.join(self.nodes[0].wallets_path, self.default_wallet_name),
-            os.path.join(self.nodes[0].wallets_path)]
+            self.nodes[0].wallets_path / self.default_wallet_name / self.wallet_data_filename,
+            self.nodes[0].wallets_path / 'x' / self.default_wallet_name / self.wallet_data_filename,
```

Comment:
the original `.` here pathlib was resolving it to remove it whereas os.path was keeping it and since it was passing as strings I mixed this up thinking we're trying to fail one of these source paths but you're right, I switched it back to `.` and it works.